raiden/tests/utils/network.py

```diff
@@ -298,3 +320,159 @@ def create_apps(
         apps.append(app)
 
     return apps
+
+
+def jsonrpc_services(
+        deploy_key,
+        deploy_client,
+        private_keys,
+        poll_timeout,
+        web3=None,
+):
+    deploy_blockchain = BlockChainService(
+        deploy_key,
+        deploy_client,
+        GAS_PRICE,
+    )
+
+    secret_registry_address = deploy_contract_web3(
```

Comment:
next step is to move the deployment of these smart contracts to separate functions and fixtures